client/src/App.js

```diff
@@ -24,17 +25,20 @@ function App() {
 
   useEffect(() => {
     async function getToken() {
-      const response = await fetch('/auth/token');
-      const json = await response.json();
-      setToken(json.access_token);
+      const response = await axios('/auth/token');
```

Comment:
Change from async function getToken to a try/catch block.

Reply:
Effect hooks are sync to prevent race conditions. Not changing to try/catch block.